docs/config.json

```diff
@@ -437,6 +437,10 @@
             {
               "title": "teleport-kube-agent",
               "slug": "/kubernetes-access/helm/reference/teleport-kube-agent/"
+            },
+            {
+              "title": "teleport-plugin-pagerduty",
+              "slug": "/kubernetes-access/helm/reference/teleport-plugin-pagerduty/"
```

Comment:
Same comment here as for the Jira one - I think we should find a better place for this in the docs.